deploy/charts/cert-manager/values.yaml

```diff
@@ -216,6 +216,26 @@ webhook:
     #   cpu: 10m
     #   memory: 32Mi
 
+  ## Liveness and readiness probe values
+  ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  ##
+  livenessProbe:
+    path: /livez
+    port: 6080
+    failureThreshold: 3
+    initialDelaySeconds: 60
+    periodSeconds: 10
+    successThreshold: 1
+    timeoutSeconds: 3
```

Comment:
I wouldn't change the defaults of these

Reply:
The only thing I changed here is the timeout. The rest of the parameters is currently used with these values by the webhook...

Reply:
Yes, sorry, I meant the timeouts.

If they are configurable then do we need to change the defaults from what has apparently worked well in the past?